Content.Server/Objectives/Commands/ListObjectivesCommand.cs

```diff
@@ -46,5 +44,17 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
             }
 
         }
+
+        public override CompletionResult GetCompletion(IConsoleShell shell, string[] args)
+        {
+            if (args.Length == 1)
+            {
+                var playerMgr = IoCManager.Resolve<IPlayerManager>();
+                var options = playerMgr.ServerSessions.Select(c => c.Name).OrderBy(c => c).ToArray();
```

Comment:
1. Put IPlayerManager as a dependency on the command itself rather than resolving.
2. Use ToList then sort rather than Select + OrderBy + ToArray.

Reply:
How do I do this?